tests/FsToolkit.ErrorHandling.Tests/Seq.fs

```diff
@@ -72,13 +70,10 @@ let traverseOptionMTests =
                     "Hola"
                 }
 
-            let expected = Seq.toList tweets
+            let expected = Seq.toArray tweets
             let actual = Seq.traverseOptionM tryTweetOption tweets
 
-            let actual =
-                Expect.wantSome actual "Expected result to be Some"
-                |> Seq.toList
```

Comment:
note IMO needing to manipulate the result to look at it is a design smell (especially when there is no actual laziness, but yet you can't be 100% that you don't need to do a Seq.cache without seeing the impl as the type system does not guarantee it)